### DISCUSSION TITLE
UTF-8 Problem for JSON Output

### DISCUSSION THREAD

**Discussion by hunkardoner** [omkarcloud/google-maps-scraper#48] [General]
When there are Turkish characters in the captured data (letters such as ö, ç, ü, ı), images such as "kadik\u00f6y" (by the way \u00f6 means ö :)) appear in the Json outputs. How can we solve this?

**Comment by Chetan11-dev** [comment]
Kindly upload the CSV to Google Sheets
![turkish-character-fix](https://github.com/omkarcloud/google-maps-scraper/assets/53407137/1362b910-7102-4c74-9231-c18cb4504161)
